pandas/tests/indexes/common.py

```diff
@@ -514,12 +514,12 @@ def test_union_base(self, indices):
 
     @pytest.mark.parametrize("sort", [None, False])
     def test_difference_base(self, sort, indices):
-        if isinstance(indices, CategoricalIndex):
-            return
-
         first = indices[2:]
         second = indices[:4]
-        answer = indices[4:]
+        if isinstance(indices, CategoricalIndex) or indices.is_boolean():
```

Comment:
Is this change related?

Reply:
Yes, it is. To make the indices fixture more useful I enhanced its length from 2 to 10 entries, which impacts the expected value here. In the same way, I could provide a proper "expected" for `CategoricalIndex` instead of just skipping it.